examples/data-objects/webflow/src/view/layout.ts

```diff
@@ -92,7 +91,7 @@ export class Layout extends EventEmitter {
     private renderPromise = done;
     private renderResolver: () => void;
 
-    constructor(public readonly doc: FlowDocument, public readonly root: Element, formatter: Readonly<RootFormatter<IFormatterState>>, public readonly viewFactoryRegistry: Map<string, IFluidHTMLViewFactory> = new Map(), public readonly scope?: FluidObject) {
+    constructor(public readonly doc: FlowDocument, public readonly root: Element, formatter: Readonly<RootFormatter<IFormatterState>>, public readonly scope?: FluidObject) {
```

Comment:
While you're here, could you also please remove `scope`?  Looks like it was only potentially used by the view factories.

Reply:
fixed in f9d718e3959316cd59736cf00d497f52330f54f8